stl/inc/chrono

```diff
@@ -609,8 +610,22 @@ namespace chrono {
         static constexpr bool is_steady = true;
 
         _NODISCARD static time_point now() noexcept { // get current time
-            const long long _Freq = _Query_perf_frequency(); // doesn't change after system boot
-            const long long _Ctr  = _Query_perf_counter();
+#if (defined(_M_ARM) || defined(_M_IX86) || defined(_M_ARM64) || defined(_M_X64)) && !defined(_M_CEE_PURE)
+            // ad-hoc <atomic> load/compare_exchange_strong with std::memory_order_relaxed

```

Comment:
This use of "atomic" as an adjective really has nothing to do with the header `<atomic>`
```suggestion
            // ad-hoc atomic load/compare_exchange_strong with std::memory_order_relaxed
```